interface/wx/spinctrl.h

```diff
@@ -189,6 +191,8 @@ class wxSpinCtrl : public wxControl
 
         @note Setting a range including negative values is silently ignored
         if current base is set to 16.
+
+        @note Backwards ranges are not supported under wxQt.
```

Comment:
I'd make it more prescriptive:
```suggestion
        @note In wxQt @a minVal must be less than @a maxVal.
```